bittensor_cli/src/bittensor/extrinsics/registration.py

```diff
@@ -737,8 +736,12 @@ async def run_faucet_extrinsic(
     # Attempt rolling registration.
     attempts = 1
     successes = 1
+    pow_result: Optional[POWSolution]
```

Comment:
why you add this if we have `pow_result=None` in line 745? 
line 745 should be more than enough since `pow_result` variable is using just within `while` loop.

Reply:
Just so mypy wouldn't yell at me on line 746 😅